src/components/dialogs/generator-creation-dialog.js

```diff
@@ -194,7 +198,6 @@ const GeneratorCreationDialog = ({
         label: 'MaximumActivePowerText',
         validation: {
             isFieldRequired: true,
-            isFieldNumeric: true,
```

Comment:
You removed isFieldNumeric from the validation of these fields because the useDoubleValue already prevent text prompts ?

Reply:
Because it's already added by the hook useDoubleValue